scripts/retrieve_block/retrieve_block.go

```diff
@@ -41,7 +42,7 @@ func buildRequestMessage(arg string) *messages.BlockRequestMessage {
 			messages.BootstrapRequestData, messages.Descending)
 	}
 
-	log.Fatalf("not supported direction: %s, use 'asc' for ascending or 'desc' for descending", params[1])
+	log.Printf("not supported direction: %s, use 'asc' for ascending or 'desc' for descending\n", params[1])
```

Comment:
nit/maybe `warnf` is appropriate here?

Reply:
I'm using `log` from golang stdlib and it does not have the Warnf method, so since it is a script log I don't find any problem in using Printf